guides/micronaut-custom-validation-annotation/micronaut-custom-validation-annotation.adoc

```diff
@@ -0,0 +1,81 @@
+common:header.adoc[]
+
+common:requirements.adoc[]
+
+common:completesolution.adoc[]
+
+common:create-app-features.adoc[]
+
+== Writing the application
+
+In this guide, you will code an annotation - `@E164` - to validate phone numbers.
+
+https://www.twilio.com/docs/glossary/what-e164[E-164]
+
+____
+E.164 is the international telephone numbering plan that ensures each device on the PSTN has globally unique number.
+
+This number allows phone calls and text messages can be correctly routed to individual phones in different countries. E.164 numbers are formatted [+] [country code] [subscriber number including area code] and can have a maximum of fifteen digits.
+____
+
+=== Country Code
+
+Create an enum for the country code. The following enum maps the https://www.itu.int/dms_pub/itu-t/opb/sp/T-SP-E.164D-11-2011-PDF-E.pdf[ ITU_T recommendation for E.164 assigned country codes].
+
+source:CountryCode[]
```

Comment:
This will render huge in the guide. Perhaps abbreviate it with some asciidoctor tags?